samples/openthread/coap_server/README.rst

```diff
@@ -71,8 +71,6 @@ Specify the corresponding file names in the :makevar:`OVERLAY_CONFIG` option whe
 See :ref:`cmake_options` for instructions on how to add this option.
 For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 
-The following configuration files are available:
```

Comment:
Why have you removed this line?

Reply:
Even though the introductory line says "The following configuration files are available:" there is only one file mentioned below. Are there any more files to be documented? If there is only file, we might not even need a bulleted list and may be we can rephrase this as ":file:`overlay-logging.conf` is a configuration file that enables logging using RTT.  For additional options, refer to :ref:`RTT logging <ug_logging_backends_rtt>`."